File: Models/Employee.cs
using System.Text.Json.Serialization;

namespace TestTask.Models
{
    public class Employee
    {
        public int Id { get; set; }

        public int DepartmentId { get; set; }
        [JsonIgnore]
        public virtual Department Department { get; set; }
        public string Name { get; set; }
        public decimal Salary { get; set; }
        public string Fruit { get; set; }
    }
}

File: ClientApp/src/app/Employee.ts
export class Employee {
    id: number;
    departmentId: number;
    name: string;
    salary: number;
    fruit: string;
}

export class ModelEmployee {
    employee: Employee;
    selected: boolean = false;

    constructor(employee: Employee) {
        this.employee = employee;
    }
}